.github/workflows/helm-update-index.yaml

```
@@ -6,7 +6,7 @@ on:
       - '**.tgz'
```

```suggestion
      - '**.tgz'
      - '.github/workflows/helm-update-index.yaml'
```
With this, the check will run if this file is changed (as in this PR).